packages/amplify_datastore/example/lib/models/TestEnum.dart

```diff
@@ -0,0 +1,18 @@
+/*
+* Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+// ignore_for_file: public_member_api_docs
+
+enum TestEnum { yes, no, maybe }
```

Comment:
Lol - reminds me of [this](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canPlayType#return_value)